app/src/main/res/menu/download_menu.xml

```diff
@@ -1,29 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<menu xmlns:android="http://schemas.android.com/apk/res/android"
+<menu xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <item android:id="@+id/switch_mode"
         android:icon="?attr/ic_grid"
         android:title="@string/grid"
+        tools:ignore="AlwaysShowAction"
         app:showAsAction="ifRoom" />
 
     <item android:id="@+id/start_downloads"
         android:visible="false"
         android:icon="?attr/play"
         android:title="@string/start_downloads"
-        app:showAsAction="ifRoom" />
+        app:showAsAction="never" />
```

Comment:
Because one of them are displayed.
In 5.1'' (or minor) screens the `grid/list` button and `start` or `pause` button are visible. I think this can confuse the user if one or more downloads are running or paused.
If you take a look at the RecyclerView adapter code, both buttons are displayed if exists running and paused downloads.

![Screenshot_1565025770](https://user-images.githubusercontent.com/16857233/62482743-65a30e00-b78c-11e9-9ae7-adeef8bc1096.png)

Reply:
> I think this can confuse the user if one or more downloads are running or paused.

That's a good point

> If you take a look at the RecyclerView adapter code, both buttons are displayed if exists running and paused downloads.

I can only see this when downloading a video
![image](https://user-images.githubusercontent.com/17365767/62528536-4b753880-b83d-11e9-87a8-fb67e12298f7.png)

Reply:
thats because the screen resolution is small, try in landscape